app/server/lib/scim/index.ts

```diff
@@ -0,0 +1,14 @@
+import * as express from 'express';
+
+import { buildScimRouterv2 } from './v2/ScimV2Api';
+import { HomeDBManager } from 'app/gen-server/lib/homedb/HomeDBManager';
+import { InstallAdmin } from '../InstallAdmin';
+
+const buildScimRouter = (dbManager: HomeDBManager, installAdmin: InstallAdmin) => {
+  const v2 = buildScimRouterv2(dbManager, installAdmin);
```

Comment:
Dumb question, but why are we versioning the API from v2? Is it because that's the version that scimmy uses?

Reply:
Not dumb any at all!

It is a recommandation of the RFC: https://www.rfc-editor.org/rfc/rfc7644#section-3.13

> The Base URL MAY be appended with a version identifier as a separate
>   segment in the URL path.  At the time of this writing, the identifier
>   is 'v2'.